community/security/src/test/java/org/neo4j/server/security/auth/AuthProceduresIT.java

```diff
@@ -92,7 +92,7 @@ public void shouldNotChangeOwnPasswordIfNewPasswordInvalid() throws Exception
     @Test
     public void shouldCreateUser() throws Exception
     {
-        assertEmpty( admin, "CALL dbms.createUser('andres', '123', true)" );
+        assertEmpty( admin, "CALL dbms.security.createUser('andres', '123', true)" );
```

Comment:
Seem to be missing tests in community for creating user with default value of changePasswordRequired